app/assets/stylesheets/application.scss

```diff
@@ -7,4 +7,9 @@
 
 // Import additional stylesheets as you make them.
 @import "video-js";
-@import "tutorials";
\ No newline at end of file
+@import "tutorials";
+@import "bootstrap.min";
+@import "bootstrap";
```

Comment:
You only need bootstrap.min–or bootstrap–but not both. They're the same. The .min version is just a compressed file.